contrib/python-cheroot/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "python-cheroot"
+pkgver = "10.0.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools_scm",
+    "python-wheel",
+]
+checkdepends = ["python-pytest"]
+pkgdesc = "Python http server"
+maintainer = "ttyyls <contact@behri.org>"
+license = "BSD-3-Clause"
+url = "https://cheroot.cherrypy.dev"
+source = f"$(PYPI_SITE)/c/cheroot/cheroot-{pkgver}.tar.gz"
+sha256 = "e0b82f797658d26b8613ec8eb563c3b08e6bd6a7921e9d5089bd1175ad1b1740"
+# failures
+options = ["!check"]
```

Comment:
then deselect the things that fail

Reply:
more like a complete failure of the cli itself

```
=> python-cheroot-10.0.1-r0: running do_check...
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: --numprocesses=auto --cov=cheroot --cov-report --cov-report xml:.test-results/pytest/cov.xml
  inifile: /builddir/python-cheroot-10.0.1/pytest.ini
  rootdir: /builddir/python-cheroot-10.0.1

=> A failure has occurred!
```

Reply:
I've tried to do a manual self.do("python", "-m", "pytest") and it also doesn't work

Reply:
that's not a complete failure, that's missing checkdepends

Reply:
pytest reads args from the pytest config file